R/optNll.R

```diff
@@ -20,8 +19,7 @@
     dataList = NULL,
     antigen_isos = data |> pull("antigen_iso") |> unique(),
     lambda.start = 1/365.25,
-    hessian = TRUE,
```

Comment:
We need `hessian = TRUE` in order to calculate standard errors, so we don't want to let the users control it. (I had previously added this parameter to the inputs without thinking it through.) Now we have hard-coded `hessian=TRUE` below (line 74).